foreman/foreman.spec

```diff
@@ -570,6 +560,7 @@ sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/se
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile RAILS_ENV=production --trace
+%{scl_rake} webpack:compile --trace
```

Comment:
We'll have to install webpack (`npm install') before running this, or put it in the /bin directory of the Foreman source tar. Any preferences? This means we have to add npm as a BuildRequires too, otherwise we'll get a `npm: command not found` error when building